packages/nimble-components/src/accordion/specs/IxD.md

```diff
@@ -0,0 +1,193 @@
+</p>
+
+Clicking anywhere on a header button opens (if closed) or closes (if opened) the associated panel of content. [**QUESTION:** Do panels only display block content (one component per "line") or do they display flex content?]
+
+<p>
+<kbd><img src="spec-images/Accordion-3.png"></kbd>
+<br>
+<sub>Click</sub>
+</p>
+
+The client user can specify whether one panel or more than one panel can be opened at a time. If the former, clicking to open a closed panel will close the open panel (if applicable).
+
+<p>
+<kbd><img src="spec-images/Accordion-4.png"></kbd>
+<br>
+<sub>Click (one panel open at a time)</sub>
+</p>
+
+If a component in a panel is in an error state, the error state is indicated on the associated header button when closed (if not also when open).
+
+<p>
+<kbd><img src="spec-images/Accordion-5.png"></kbd>
+<br>
+<sub>Error</sub>
+</p>
+
+### Additional Options
+
+Client users may need/want to have additional buttons and/or icons in the header.
+
+<p>
+<kbd><img src="spec-images/Accordion-6.png"></kbd>
+<br>
+<sub>"Expand" button</sub>
+</p>
+
+### Nested Accordions
+
+Accordion panels can have one level of nested accordions. Nested accordions are always ghost style. [**QUESTION:** Can accordions be nested anywhere in the panel? Can there be multiple child accordions nested in an accordion?]
+
+<p>
+<kbd><img src="spec-images/Accordion-7.png"></kbd>
+<br>
+<sub>Nested accordion in "info" panel</sub>
+</p>
+
+### Keyboard Interactions
+
+| Key                | Description                                           |
+| ------------------ | ----------------------------------------------------- |
+| `TAB`              | Moves focus to next focusable element                 |
+| `SPACE` or `ENTER` | Opens/closes accordion section when header is focused |
+
+`TAB` to the accordion focuses the first header button (if not also visually indicating the entire accordion).
+
+<p>
+<kbd><img src="spec-images/Accordion-8.png"></kbd>
+<br>
+<sub>TAB to accordion</sub>
+</p>
+
+`TAB` focuses the next focusable element (if not also visually indicating the header button and/or the entire accordion). All focusable elements in the accordion are included in the `TAB` sequence.
+
+<p>
+<kbd><img src="spec-images/Accordion-9.png"></kbd>
+<br>
+<sub>TAB to focusable element in accordion panel</sub>
+</p>
+<p>
+<kbd><img src="spec-images/Accordion-10.png"></kbd>
+<br>
+<sub>TAB to next header button in accordion</sub>
+</p>
+
+`ENTER` or `SPACE` on a focused header button for a closed panel opens the associated panel. If only one panel can be opened at a time and another panel is open, it collapses.
+
+<p>
+<kbd><img src="spec-images/Accordion-11.png"></kbd>
+<br>
+<sub>ENTER to open a closed section</sub>
+</p>
+
+`ENTER` or `SPACE` on a focused header button for an open panel closes the associated panel.
+
+<p>
+<kbd><img src="spec-images/Accordion-12.png"></kbd>
+<br>
+<sub>ENTER to close an opened section</sub>
+</p>
+
+## Validation
+
+TBD
```

Comment:
The document is being provided on the document in GitHub, so I wasn't sure what to link. These specs are meant to be quick. I think ideally I make a post soliciting feedback from stakeholders on my PRs on Teams (as in person meetings could be scheduled as needed).

Likewise, engaging with user research is necessary, but 1) it seems like it would be a block to posting the spec, 2) if I'm responsible for making this happen for every component I think we need to meet with user research and develop a low-effort protocol for doing this, and 3) it seems like user testing should happen with the visual design as well?

So I'm not really sure.